src/app/components/Home/index.tsx

```diff
@@ -42,16 +47,12 @@ const Home: React.FC = () => {
 
   return (
     <React.Fragment>
-      {!user ? (
+      {loading ? (
+        <Loader />
+      ) : !user && !officesData ? (
         <Redirect to="/auth" />
       ) : (
-        <>
-          {officesData ? (
-            <GroupSelects officesProps={officesData} />
-          ) : (
-            <Loader />
-          )}
-        </>
+        <GroupSelects officesProps={officesData} />
       )}
     </React.Fragment>
```

Comment:
you only need to change redirection condition
`!user && !loading` --> redirect to `/uath`;
Also you don't need `Fragment` here;